saisdkdump/Makefile.am

```diff
@@ -8,11 +8,16 @@ else
 DBGFLAGS = -g
 endif
 
+
 if SAIVS
 SAILIB=-L$(top_srcdir)/vslib/src/.libs -lsaivs
 else
+if sonic_asic_platform_barefoot
+SAILIB=-L/opt/bfn/install/lib -lswitchsai -lswitchapi -lbf_switchd_lib -ltofinopdfixed_thrift -ldriver -lbfutils -lbfsys -lbfutils -L/opt/bfn/install/lib/tofinopd/switch -lpd -lpdcli -lpdthrift
```

Comment:
remove library other than the sai package.

Reply:
fixed